html/index.html

```diff
@@ -647,38 +666,59 @@ <h1>Sampling and Sequencing Simulator</h1>
         const n_sick = poisson(n_sample_population * individual_probability_sick);
         site_infos[site].sample_sick += n_sick;
         site_infos[site].sample_total += n_sample_population;
+
+        // Compute weekly incidence from daily incidence
+        let weekly_incidence = 0;
+        effective_incidence = daily_incidence;
+        for (let i = 0; i < 7; i++) {
+           weekly_incidence += effective_incidence;
+           effective_incidence /= growth_factor;
```

Comment:
I don't understand this block. Why isn't the weekly incidence the sum of the daily incidence over the week?

Reply:
> Why isn't the weekly incidence the sum of the daily incidence over the week?

That is what this block is calculating, yes.  Weekly incidence is the sum of how many people became sick (daily incidence) on each of the previous seven days.